src/sentry/models/release.py

```diff
@@ -363,6 +363,10 @@ def _get_cache_key(project_id: int, group_id: int, first: bool) -> str:
 
 
 class ReleaseModelManager(BaseManager["Release"]):
+    def create(self, **kwargs):
```

Comment:
Hooking into the release model manager create since we have a few api's that create releases